### PR TITLE
Break out bounties section

### PR DESCRIPTION
**Pull request by ChrisChinchilla**
Part of #12 

This doesn't include creating new links etc, just the separation.

### CODE REVIEW

File: bounties.md [hunk 1]
```diff
@@ -1,13 +1,51 @@
...
+-   **Open Track Prize:** This is the main prize of ETHBerlin. There will be 10 winners. The grand bounty is of approximately TBD € and it’s distributed equally. You can calculate it ;)
+-   **Sponsor Prizes:** Some of ETHBerlin’s sponsors have their own price awards. They can each pay out a maximum of TBD €. These prizes will be funded and distributed via Gitcoin. Note that ETHBerlin has no influence on neither the challenge nor the judging, this is entirely up to the sponsors. You can find all the prize awards [here](https://github.com/ethberlinzwei/Bounties/issues).
+-   **Community Improvement Awards:** These are small prizes endowed with TBD € each by ETHBerlin. Check them out [here](https://github.com/ethberlinzwei/Bounties).
 
-## Setting up bounties
+**IMPORTANT:** You can submit your work to multiple prize awards, they are not exclusive! For example you can submit your work to the open track bounty and a sponsor specific prize, just make sure to submit it everywhere where you want to participate.
```
Comment (ligi) [inline bounties.md:12]: I think it is "submit for awards" not "submit to awards"

Comment (ChrisChinchilla) [reply bounties.md:12]: Nope, "submit to is correct", you are submitting to the process, or something like that…

**Comment by ChrisChinchilla** [comment]
I'll handle the merge here, it's going to be a little messy

File: bounties.md (hunk 1 above)
Comment (ligi) [reply bounties.md:12]: yea - but "prize awards" is not a process as far as I know - how about: "submit your work to judgement over multiple prize awards"?

**Pull request merged**